Library/Homebrew/utils/github.rb

```diff
@@ -394,6 +394,52 @@ def search(entity, *queries, **qualifiers)
     open_api(uri) { |json| json.fetch("items", []) }
   end
 
+  def approved_reviews(user, repo, pr, commit: nil)
+    url = "https://api.github.com/graphql"
+    data = {
+      query: <<~EOS,
+        { repository(name: "#{repo}", owner: "#{user}") {
+            pullRequest(number: #{pr}) {
+              reviews(states: APPROVED, first: 100) {
+                nodes {
+                  author {
+                    ... on User { email login name databaseId }
+                    ... on Organization { email login name databaseId }
+                  }
+                  authorAssociation
+                  commit { oid }
+                }
+              }
+            }
+          }
+        }
+      EOS
+    }
+    result = open_api(url, data: data, request_method: "POST")
+    raise Error, result["errors"] if result["errors"].present?
+
+    reviews = result["data"]["repository"]["pullRequest"]["reviews"]["nodes"]
+
+    reviews.map do |r|
+      next if commit.present? && commit != r["commit"]["oid"]
+      next unless %w[MEMBER OWNER].include? r["authorAssociation"]
+
+      email = if r["author"]["email"].empty?
```

Comment:
```suggestion
      email = if r["author"]["email"].blank?
```

Just to handle that empty string case.